activejob/test/cases/delayed_job_adapter_test.rb

```diff
@@ -3,6 +3,10 @@
 require "active_job/queue_adapters/delayed_job_adapter"
```

Comment:
The reason I did file exclusion instead of skipping was to prevent loading `DelayedJob` when running tests for other adapters (mostly because `DelayedJob` prints a bunch of warnings on require)

Especially for the `isolated` tests, I also think its wrong to be loading unused files/libraries when the suite claims to be `isolated`

Reply:
Okay that's fair. I'll change it back. Thanks for the note.